packages/commons/src/fhir/index.ts

```diff
@@ -401,3 +401,29 @@ export function toHistoryResource<T extends Saved<Resource>>(resource: T) {
     : never
 }
 export { updateFHIRBundle, buildFHIRBundle } from './transformers'
+
+export function removeDuplicatesFromComposition(
+  composition: Composition,
+  duplicateId?: string
+) {
+  const compositionId = composition.id
+  if (duplicateId) {
+    const removeAllDuplicates = compositionId === duplicateId
+    const updatedRelatesTo =
+      composition.relatesTo &&
+      composition.relatesTo.filter((relatesTo) => {
+        return (
+          relatesTo.code !== 'duplicate' ||
+          (!removeAllDuplicates &&
+            relatesTo.targetReference &&
+            relatesTo.targetReference.reference !==
+              `Composition/${duplicateId}`)
+        )
+      })
+    composition.relatesTo = updatedRelatesTo
+    return composition
+  } else {
+    composition.relatesTo = []
+    return composition
+  }
+}
```

Comment:
Is this function needed in more than one microservice? If not then we can just move it to where it's being used